src/controllers/monitorController.js

```diff
@@ -0,0 +1,35 @@
+import dbQuery from '../db/config.js';
+import { nanoid } from 'nanoid';
+import { queryGetAllMonitors, queryAddMonitor } from '../db/queries.js';
+
+const getMonitors = async (req, res) => {
+  try {
+    const response = await dbQuery(queryGetAllMonitors);
```

Comment:
Think it makes more sense to have the db/queries file call dbQuery directly. Then instead of importing from both files we can do:
- import db from '../db/queries.js';
- await db.getAllMonitors();